views/user.py

```diff
@@ -11,6 +12,21 @@ def non_empty_str(val, name):
     return str(val)
 
 
+def valid_email(email):
+    regex = re.compile('[a-z]{3,}(?P<at>@)[a-z]{3,}(?P<point>\.)[a-z]{2,}')
```

Comment:
se non salvi la regexp compilata in una variabile a livello di modulo usare la compile è solo controproducente